src/tests/Loader/classloader/MethodImpl/CovariantReturns/UnitTest/UnitTestMultiModule.ilproj

```diff
@@ -1,6 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk.IL">
   <PropertyGroup>
     <OutputType>Exe</OutputType>
+    <MonoAotIncompatible>true</MonoAotIncompatible>
```

Comment:
```suggestion
```

Marking this as incompatible is what is causing the test failures. We'll revisit how to handle this better later.